### ISSUE TITLE
dependency on ess

### ISSUE THREAD

**Issue by dankessler** [polymode/poly-R#41]
`poly-R` will not load without `ess`: see [these lines](https://github.com/polymode/poly-R/blob/8024e852cfca642dea2045a41b2033baa2f1f9a5/poly-R.el#L39-L40)

However, `ess` is [not declared as a dependency](https://github.com/polymode/poly-R/blob/8024e852cfca642dea2045a41b2033baa2f1f9a5/poly-R.el#L7). Thus, it's possible to install `poly-R` *without* `ess`, in which case things won't work.  For example, if I start a fresh emacs (with `emacs -Q`), configure melpa, install `poly-R`, and then `(require 'poly-R`), I get an error about being unable to load `ess-mode`.

This bug probably rarely comes up, since most people using `poly-R` have probably already installed `ess`, but it would be nice to make this explicit/automatic.

Fixing this is as simple as making the dependency explicit, but rather than submitting a PR, I wanted to check if there's a particular version of `ess` on which `poly-R` depends.

**Comment by mmaechler** [comment]
I tend to agree with @dankessler .. and propose to add such a dependency.   
(It will have the advantage for users that they only need to install `poly-R` and that would ensure to get `ess`)

**Comment by rsparapa** [comment]
I think this means hacking line 7 of poly-R.el…

    ;; Package-Requires: ((emacs "25") (polymode "0.2.2") (poly-markdown "0.2.2") (poly-noweb "0.2.2"))

To make that…

    ;; Package-Requires: ((emacs "25") (ess "25") (polymode "0.2.2") (poly-markdown "0.2.2") (poly-noweb "0.2.2"))

 Not sure if this will fix it.  Since ESS v20250110.1437 would still satisfy (ess "25"), right?

 I guess we won’t know until we actually try it.  Just pushed it...
